ios/brave-ios/fastlane/Fastfile

```diff
@@ -64,112 +35,109 @@ platform :ios do
+          "com.brave.ios.browser.beta.BraveWireGuard" => "Brave iOS Beta Wireguard Extension",
+        },
       }
     }
-    testflight_build({overrideParams: overrideParams, skip_upload: options[:skip_upload]})
+    testflight_build({gymOverrides: gymOverrides})
   end
 
-  desc "Creates a Brave Internal Beta Release build for TestFlight."
-  lane :internal do |options|
-
-    overrideParams = {
-      scheme: "Dev",
+  desc "Creates a Brave Nightly build for TestFlight."
+  lane :nightly do |options|
+    gymOverrides = {
+      scheme: "Nightly",
       export_method: "app-store",
       export_options: {
-        manageAppVersionAndBuildNumber: false
+        manageAppVersionAndBuildNumber: false,
+        provisioningProfiles: {
+          "com.brave.ios.BrowserBeta" => "Brave iOS Nightly",
+          "com.brave.ios.BrowserBeta.ShareExtension" => "Brave iOS Nightly Share Extension",
+          "com.brave.ios.BrowserBeta.BrowserIntents" => "Brave iOS Nightly Intents Extension",
+          "com.brave.ios.BrowserBeta.BraveWidgetsExtension" => "Brave iOS Nightly Widgets Extension",
+          "com.brave.ios.BrowserBeta.BraveWireGuard" => "Brave iOS Nightly Wireguard Extension",
+        },
       }
     }
-    testflight_build({overrideParams: overrideParams, skip_upload: options[:skip_upload]})
+    testflight_build({gymOverrides: gymOverrides})
   end
 
   desc "Create an archive to be uploaded to the App Store"
   lane :release do |options|
     ENV["BRAVE_APPSTORE_BUILD"] = "1"
-    overrideParams = {
+    gymOverrides = {
       scheme: "Release (AppStore)",
       export_options: {
         method: "app-store",
-        provisioningProfiles: { 
-          "com.brave.ios.browser" => "Brave iOS",
-          "com.brave.ios.browser.ShareExtension" => "Brave iOS Share Extension",
-          "com.brave.ios.browser.BrowserIntents" => "Brave iOS Intents Extension",
-          "com.brave.ios.browser.BraveWidgetsExtension" => "Brave iOS Widgets Extension",
-          "com.brave.ios.browser.BraveWireGuard" => "Brave iOS WireGuard Extension",
+        provisioningProfiles: {
+          "com.brave.ios.browser" => "Brave iOS Release",
+          "com.brave.ios.browser.ShareExtension" => "Brave iOS Release Share Extension",
+          "com.brave.ios.browser.BrowserIntents" => "Brave iOS Release Intents Extension",
+          "com.brave.ios.browser.BraveWidgetsExtension" => "Brave iOS Release Widgets Extension",
+          "com.brave.ios.browser.BraveWireGuard" => "Brave iOS Release WireGuard Extension",
         },
         manageAppVersionAndBuildNumber: false
       },
     }
-    testflight_build({overrideParams: overrideParams, skip_upload: options[:skip_upload]})
+    testflight_build({gymOverrides: gymOverrides})
+  end
+
+  desc "Uploads a build to TestFlight"
+  lane :upload do |options|
+    api_key = app_store_connect_api_key()
+    pilotParams = {
+      api_key: api_key,
+      changelog: "Bug fixes & improvements",
```

Comment:
So we upload to testers automatically always with this testers notes? not for now but we can improve on it, maybe fetch some commit names from git

Reply:
Yes for now I chose this, but we can find a way to pass in info from Jenkins if possible. They already technically have a changelog there, but maybe we can somehow generate a changelog based on files changed in the `ios/` directory instead